modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js

```diff
@@ -84,9 +89,7 @@ const InputQuantitySelector = forwardRef(
 				quantity,
 				unitOfMeasure ? min : Math.ceil(min),
 				unitOfMeasure ? max : Math.ceil(max),
-				unitOfMeasure
-					? unitOfMeasure.incrementalOrderQuantity
-					: Math.ceil(step)
+				unitOfMeasure ? step : Math.ceil(step)
```

Comment:
why in this case we are using only the step and not the getMultiple(....) ?
I mean... we replaced the `unitOfMeasure.incrementalOrderQuantity` with `getMultiple(....` in the other places but here we completely removed it to use step

Reply:
not sure I've just refactored the code I saw from the other PR, will look at this